apcd-cms/src/apps/exception/templates/exception_submission_form/exception_threshold_form.html

```diff
@@ -230,6 +230,19 @@ <h4>Acknowledgment of Terms</h4>
                   $('#expiration-date').attr('max', maxDate);
               });
               </script>
+              <!-- To fetch CDL codes on file type input change -->
+              <script>
+                const fileTypeSelect = document.querySelector('#file_type');
+                const form = document.getElementById('threshold-form');
+                fileTypeSelect.addEventListener('change', function(event) {
+                  selectedFileType = event.target.value;
+                  if (selectedFileType != '') {
+                    const xhr = new XMLHttpRequest();
+                    xhr.open('GET', `/submissions/threshold-exception?file_type=${selectedFileType}`);
+                    xhr.send();
+                  }
+                });
+              </script>
           </aside>
```

Comment:
I think another event listener for the CDL dropdown would be good to add here - this event listener would set the max value of the threshold input element to the corresponding CDL code's max threshold. I think this would be an efficient way to use the threshold value you are reading from the db, and match the help text of that threshold input field.

Reply:
This is a great idea! I've been hacking at it and can get the field_code in a get request through a script and passed to the view, but am a little upside down on if the url changes to the field_code rather than the file_type, if the field code drop down will break. Adding the field_code after the file_type in the get request seems a little tricky too in terms of the event listeners. Maybe I can add this as a follow on task and get this drop down up for now? Also open to any suggestions or pairing up. I am so close to getting the threshold value to populate dynamically in the help text I must just be missing something here.

Reply:
Done and ready to review. Thank you!